### DISCUSSION TITLE
Question on GO panel integrated inside main GO window

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#1021] [General]
Looking at website mps-orgelsite.de i found a ùn odf downloadable sample named sp giga. It's a demo of multiple demo organs afaik.
Something caught my attention: it seems that panel is integrated in GO window (pls look at attached image).
Is it true or, as i think, it's just a collage of images ?
thanks.

![giga](https://user-images.githubusercontent.com/50132661/153211698-be63031d-f42d-4295-8231-673e6537e06a.jpg)

**Comment by oleg68** [comment]
An old version of GrandOrgue had the single windows/

**Reply by paolo2504** [reply]
Nice. Thank you.
